roles/core/pxe_stack/defaults/main.yml

```diff
@@ -1,3 +1,4 @@
 ---
 pxe_stack_enable_cloning: false
 pxe_stack_firewall_zone:
+pxe_stack_sshd_force_PermitRootLogin: true
```

Comment:
why the `sshd_force_PermitRootLogin` is not an option on ssh_master configuration ?

Reply:
This parameter ensure you can ssh as root on the node after it has been deployed.
Recent versions of Fedora, and RHEL 9 do not allow ssh as root by default. So it is more during PXE 🙂

Reply:
ok, i didn't have this info, i'm ok then; )